src/openforms/js/components/admin/form_design/Confirmation.js

```diff
@@ -42,6 +42,7 @@ const EMAIL_OPTIONS = [
 
 const Confirmation = ({
   displayMainWebsiteLink = false,
+  includeConfirmationPageContentInPdf: includeConfirmationPageContentInPdf = false,
```

Comment:
```suggestion
  includeConfirmationPageContentInPdf = false,
```
not sure why you are aliasing the prop to itself